docs/self-managed/setup/deploy/local/docker-compose.md

```diff
@@ -1,26 +1,254 @@
 ---
 id: docker-compose
 title: "Docker Compose"
-keywords: ["camunda docker"]
+keywords: ["camunda docker", "docker compose"]
 ---
 
-A Docker Compose configuration to run Zeebe, Operate, Tasklist, Optimize, Identity, and Connectors Bundle is available in the [camunda-platform](https://github.com/camunda/camunda-platform/blob/main/docker-compose.yaml) repository.
-Follow the instructions in the [README](https://github.com/camunda/camunda-platform#using-docker-compose).
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
 
-:::warning
-While the [Docker images](/self-managed/setup/deploy/other/docker.md) themselves are supported for production usage, the Docker Compose files are designed to be used by developers to run an environment locally; they are not designed to be used in production. We recommend to use [Kubernetes](/self-managed/setup/install.md) in production.
+A Docker Compose configuration to run Camunda Self-Managed components (Zeebe, Operate, Tasklist, Optimize, Identity, and Connectors).
+
+:::danger
+While the [Docker images](/self-managed/setup/deploy/other/docker.md) themselves are supported for production usage, the Docker Compose files are designed to be used by developers to run an environment locally, and are not designed to be used in production. We recommend [Kubernetes](/self-managed/setup/install.md) for production use cases.
+:::
+
+## Prerequisites
+
+- **Docker Compose:** Version 1.27.0+ (supports the [latest compose specification](https://docs.docker.com/compose/compose-file/)).
+- **Docker:** Version 20.10.16+
+- **Keycloak:** (Local development only). Add Keycloak to resolve to 127.0.0.1 on your local machine, and set `KEYCLOAK_HOST=keycloak` in the `.env` file for token refresh and logout functionality.
+
+## Run Camunda 8 with Docker Compose
+
+To start a complete instance of Camunda 8 Self-Managed environment locally:
+
+1. Clone the [Camunda Self-Managed repository](https://github.com/camunda/camunda-self-managed), and navigate to the directory of the desired Camunda version (for example, `/version/camunda-8.6`).
+2. Run the following command:
+
+```shell
+docker compose up -d
```

Comment:
Pending the resolution of the profile question - right now, this needs to be run with `--profile full`, or the compose file needs to be updated to remove that requirement.

Reply:
I've updated the SM repo and removed the profiles as they are not used any more.